numpy/matlabequivalent.py

```diff
@@ -0,0 +1,11 @@
+import numpy as np
+
+#contribution no.1      08/07/2019
+#matlab equivalent of bitget
+def bitget(value,bit_no):                  # equivalent bitget in matlab!!
+    binval=eval(np.binary_repr(value))de
```

Comment:
Typo here.

Also, as a general rule, the use of `eval` should raise red flags - even if it's not a security issue (in this case, it's safe), its often a symptom of there being a better way to solve the problem

Reply:
Sorry for the typo! but, there is no such function getbit in Numpy/python as there is in Matlab! 
Should I send it to the mailing list? I am new to the open-source!